cleanlab/object_detection/rank.py

```diff
@@ -66,6 +67,7 @@ def get_label_quality_scores(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
     aggregation_weights: Optional[Dict[str, float]] = None,
```

Comment:
This `aggregation_weights` argument should also come after the * (as with all optional arguments).

@ulya-tkch why was this argument put before the *

Reply:
I did not add `aggregation_weights` param to the repo, but I should've caught that during this review. Good point!